helm-chart/renku/values.yaml

```diff
@@ -5,6 +5,8 @@
 ## Global variables
 ## Shared values/secrets
 global:
+  platformConfig: |
+    {} 
```

Comment:
I see that the `platform-init` script uses json to parse this. Lets switch to yaml. It is just nicer to write it and it is nicer to have yaml in a file that already has a bunch of yaml like the values file.